### PR TITLE
add manifest status in command `agent status` output

### PR DESCRIPTION
**Pull request by kangyili**
### What does this PR do?

The output now contains manifest status
```
=========
Forwarder
=========

  Transactions
  ============
.....
    OrchestratorManifest: 1
....

```

```
  ===========
  Manifest Buffer Stats
  ===========
  Buffer Flushed : 4 times
  Last Time Flushed Manifests : 1
  ==================================================
  Manifests Flushed Per Resource
  ==================================================
    ClusterRole : 3
    ClusterRoleBinding : 3
    CronJob : 0
    CustomResource : 0
    CustomResourceDefinition : 0
    DaemonSet : 1
    Deployment : 2
    Ingress : 0
    Job : 0
    Namespace : 7
    Node : 1
    PersistentVolume : 0
    PersistentVolumeClaim : 0
    Pod : 0
    ReplicaSet : 3
    Role : 0
    RoleBinding : 0
    Service : 2
    ServiceAccount : 37
    StatefulSet : 0
    VerticalPodAutoscaler : 0

```

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/collector/corechecks/cluster/orchestrator/manifest_buffer.go [hunk 1]
```diff
@@ -137,3 +152,19 @@ func BufferManifestProcessResult(messages []model.MessageBody, buffer *ManifestB
...
 }
+
+func setManifestStats(manifests []interface{}) {
+	// Number of manifests flushed
+	manifestFlushed.Set(int64(len(manifests)))
+	// Number of times the buffer is flushed
+	bufferFlushedTotal.Add(1)
+	// Number of manifests flushed per resource in total
+	for _, m := range manifests {
+		nodeTye := orchestrator.NodeType(m.(*model.Manifest).Type)
```
Comment (fisherevans) [inline pkg/collector/corechecks/cluster/orchestrator/manifest_buffer.go:163]: nit: typo

**Review by fisherevans — APPROVED** [review]
one nit

File: pkg/status/templates/orchestrator.tmpl [hunk 2]
```diff
@@ -54,6 +57,19 @@ Orchestrator Explorer
     {{ $element.NodeType }}
       Last Run: (Hits: {{$element.CacheHits}} Miss: {{$element.CacheMiss}}) | Total: (Hits: {{$element.TotalHits}} Miss: {{$element.TotalMiss}})
 {{end}}
+{{- if .ManifestCollection }}
+  ===========
+  Manifest Buffer Stats
+  ===========
```
Comment (Kaderinho) [inline pkg/status/templates/orchestrator.tmpl:63]:
```suggestion
  =====================
  Manifest Buffer Stats
  =====================
```

File: pkg/status/templates/orchestrator.tmpl [hunk 3]
```diff
@@ -54,6 +57,19 @@ Orchestrator Explorer
...
+  Buffer Flushed : {{.BufferFlushed}} times
+  Last Time Flushed Manifests : {{.ManifestsFlushedLastTime}}
+  ==================================================
+  Manifests Flushed Per Resource
+  ==================================================
```
Comment (Kaderinho) [inline pkg/status/templates/orchestrator.tmpl:68]:
```suggestion
  ==============================
  Manifests Flushed Per Resource
  ==============================
```

**Review by Kaderinho — APPROVED** [review]
Small format nits (I believe it's better for consistency and readability but feel free to ignore if you disagree)

**Pull request merged**